hrui/css/hrui.css

```diff
@@ -5,3 +5,33 @@
 #crm-main-content-wrapper #contactDetails #govID span {
     margin-right: 1%;
 }
+
+#crm-container .crm-contact-page {
+    position: relative;
+    padding: 4px;
+}
+#crm-container #mainTabContainer {
+    background-color: transparent;
+    padding: 0px;
+}
+
+#mainTabContainer .ui-tabs-nav {
+    width: 15%;
+    float:left;
+    padding:4px;
+}
+#mainTabContainer li.crm-tab-button {
+    display: block;
+    width: 100%;
+    font-size: 12px;
+    white-space: normal;
+}
+
+#crm-container .ui-tabs-panel {
+    float: right;
+    width: 83%;
+    background-color: #FFF;
+}
+
+.ui-widget-content { background: transparent; } /* override of jquery-ui.css that creates weird white background */
+.ui-dialog { background: #FFF; }
```

Comment:
There should be a newline at the end of every file. Make sure your IDE can use the [.editoconfig|https://github.com/civicrm/civihr/blob/develop/.editorconfig] file that we have in the repo